main.cpp

```diff
@@ -139,18 +142,18 @@ float Car::calculateTrueSellValue(std::string carMake, float carSellingPrice, in
     return std::abs(trueSellValue);
 }
 
-int Car::calculateMilesUntilOilChange(int fuelTypeCar, int totalMiles)
+void Car::calculateMilesUntilOilChange(int fuelTypeCar, int totalMiles)
 {
     int milesUntilOilChange = 0;
 
     if (!(milesUntilOilChange = totalMiles % 5000))
     {
         std::cout << fuelTypeCar << " car needs doesn't need oil change" << std::endl;
-        return milesUntilOilChange;
+        return;
     }
 
     std::cout << "Car needs oil change in " << milesUntilOilChange << " miles"<< std::endl;
-    return milesUntilOilChange;                                       
+    return;                                     
 }
```

Comment:
Functions that don't return anything don't need `return;` written at the end
- [ ] revise

Reply:
In this case I actually want it to return from the void function because I don't want it to output the last std::cout only the one in the if statement. But since if else is redundant for this not sure how to go about it

Reply:
In this case,  I do want the void function to return from the function to avoid it printing out the std::cout outside of the if statement.